backend/canisters/translations/impl/src/model/translations.rs

```diff
@@ -162,6 +168,47 @@ impl Translations {
             .collect()
     }
 
+    pub fn collate_decision_summaries(&self, since: TimestampMillis) -> HashMap<UserId, DecisionSummary> {
+        self.translations
+            .iter()
+            .filter(|t| match &t.status {
+                TranslationStatus::Approved(s) => s.attribution.when > since,
+                TranslationStatus::Rejected(s) => s.attribution.when > since,
+                TranslationStatus::Deployed(s) => s.deployed > since,
+                _ => false,
+            })
+            .group_by(|t| t.proposed.who)
```

Comment:
This will only group consecutive values, so probably isn't what you want.

Reply:
I've just seen this which may work here - https://docs.rs/itertools/0.8.2/itertools/trait.Itertools.html#method.into_group_map

Reply:
Ah thanks. Missed that